data/categories.txt

```diff
@@ -14991,3 +14991,178 @@ uk:Електростанція
 vi:Nhà máy điện
 zh-Hans:发电厂
 zh-Hant:發電廠
+
+Generator|@generator
+en : Generator
+de : Generator
+es : Generador
+et : Generaator
+it : Generatore
```

Comment:
Why did you insert spaces?

Reply:
oh..k sir i will remove those spaces and again give the pull request
Thank you for reviewing..

Reply:
> oh..k sir i will remove those spaces and again give the pull request Thank you for reviewing..

@biodranik sir anywhere else any input needed please tell me...or just the problem of spaces is there..

Reply:
@biodranik sir i have pushed the code with changes and with some other fixes please review it...
Regards,
N DIVIJ